lib/pycoproc/pycoproc.py

```diff
@@ -81,14 +87,15 @@ class Pycoproc:
 
     EXP_RTC_PERIOD = const(7000)
 
-    def __init__(self, i2c=None, sda='P22', scl='P21'):
+    def __init__(self, i2c=None, sda='P22', scl='P21', board_type = PYTRACK):
         if i2c is not None:
             self.i2c = i2c
         else:
-            self.i2c = I2C(0, mode=I2C.MASTER, pins=(sda, scl), baudrate=100000)
+            self.i2c = I2C(0, mode=I2C.MASTER, pins=(sda, scl))
```

Comment:
why are we removing the baudrate?

Reply:
I added it a while ago to try and fix an issue but it didn't change anything